src/components/FeatureViews/Scheduler/SessionView.js

```diff
@@ -1,154 +1,389 @@
+                  }
+                  session={session}
+                  setPos
+                />
+              )}
+            </Grid>
+          </Grid>
+        </Grid> */}
+        <Grid
+          align="left"
+          className="session-view-details"
+          container
+          item
+          spacing={2}
+          xs={6}
+        >
+          <Grid item xs={6}>
+            <Typography variant="h5">Subject</Typography>
+            <Typography>{courseCategory.name}</Typography>
+          </Grid>
+          <Grid item xs={6}>
+            <Typography variant="h5">Room</Typography>
+            <Typography>{room || "TBA"}</Typography>
+          </Grid>
+          <Grid item xs={12}>
+            <Typography variant="h5">
+              Instructor
+              {confirmed ? (
+                <ConfirmIcon className="confirmed course-icon" />
+              ) : (
+                <UnconfirmIcon className="unconfirmed course-icon" />
+              )}
+            </Typography>
+            {course && (
+              <NavLink
+                style={{ textDecoration: "none" }}
+                to={`/accounts/instructor/${instructor.user.id}`}
+              >
+                <Tooltip
+                  aria-label="Instructor Name"
+                  title={fullName(instructor.user)}
+                >
+                  <Avatar style={styles(fullName(instructor.user))}>
+                    {fullName(instructor.user)
+                      .match(/\b(\w)/g)
+                      .join("")}
+                  </Avatar>
+                </Tooltip>
+              </NavLink>
+            )}
+          </Grid>
+          <Grid item xs={12}>
+            <Typography align="left" variant="h5">
+              Students Enrolled
+            </Typography>
+            <Grid container direction="row">
+              {enrollmentSet.length > 0 ? (
+                enrollmentSet.map((student) => (
+                  <NavLink
+                    key={student.student.user.id}
+                    style={{ textDecoration: "none" }}
+                    to={`/accounts/student/${student.student.user.id}/${course_id}`}
+                  >
+                    <Tooltip title={fullName(student.student.user)}>
+                      <Avatar style={styles(fullName(student.student.user))}>
+                        {fullName(student.student.user)
+                          .match(/\b(\w)/g)
+                          .join("")}
+                      </Avatar>
+                    </Tooltip>
+                  </NavLink>
+                ))
+              ) : (
+                <Typography variant="body">
+                  No students enrolled yet.
+                </Typography>
+              )}
+            </Grid>
+          </Grid>
+          <Grid item xs={6}>
+            <Typography variant="h5">Day(s)</Typography>
+            <Typography>
+              {availabilityList.map(({ dayOfWeek }, index) => {
```

Comment:
use Andrew's new component

Reply:
So Andrew's component renders the time as T/TH, 10:00am - 11:00am. In this component you have a section for Day(s) and Time. Please see below. So for components that separate the days and time I used the functions within the component. Should I update the stylings to use Andrew's component? Or should I leave as is?
<img width="1919" alt="Screen Shot 2020-12-01 at 5 49 26 PM" src="https://user-images.githubusercontent.com/61895571/101711997-3eccaa00-3a5a-11eb-908f-6c59ab8ae44e.png">

Reply:
Note: this is an earlier image of this component, under days it would now say T/Th

Reply:
hm. this is a good question actually. @katiebelleho what do you think?

Reply:
This was resolved in teams, with keeping the stylings as is.